packages/cysync-core/src/dialogs/Send/Dialogs/Components/AddressAndAmountSection/index.tsx

```diff
@@ -1,12 +1,14 @@
-import { TabContentContainer, Tabs } from '@cypherock/cysync-ui';
+import { CoinFamily } from '@cypherock/coins';
+import { Container, TabContentContainer, Tabs } from '@cypherock/cysync-ui';
 import React from 'react';
 
+import { useSendDialog } from '~/dialogs/Send/context';
 import { selectLanguage, useAppSelector } from '~/store';
 
-import { BatchTransactionBody } from './BatchTransactionBody';
+import { BatchTransaction } from './BatchTransaction';
 import { SingleTransaction } from './SingleTransaction';
 
-export const AddressAndAmountSection: React.FC = () => {
+const BitcoinAnA: React.FC = () => {
```

Comment:
Update this for other variables as well
```suggestion
const BitcoinAddressnAmount: React.FC = () => {
```

Reply:
renamed